app/controllers/facility_reservations_controller.rb

```diff
@@ -229,7 +228,7 @@ def sort_lookup_hash
       "reserve_range" => "CONCAT(reservations.reserve_start_at, reservations.reserve_end_at)",
       "product_name"  => "products.name",
       "status"        => "order_statuses.name",
-      "assigned_to"   => "CONCAT(assigned_users_order_details.last_name, assigned_users_order_details.first_name)",
+      "assigned_to"   => "CONCAT(assigned_users.last_name, assigned_users.first_name)",
```

Comment:
It might be nice to make the reservations and orders controllers a little more consistent by using the same format, either `CONCAT` or the array. I think I lean towards the array.